src/viewport.cpp

```diff
@@ -177,31 +180,87 @@ struct ViewportDrawer {
 	Point foundation_offset[FOUNDATION_PART_END];    ///< Pixel offset for ground sprites on the foundations.
 };
 
-static void MarkViewportDirty(const ViewPort *vp, int left, int top, int right, int bottom);
+static void MarkViewportDirty(Viewport * const vp, int left, int top, int right, int bottom);
 
 static ViewportDrawer _vd;
 
+uint _vp_route_step_width = 0;
+uint _vp_route_step_height_top = 0;
+uint _vp_route_step_height_middle = 0;
+uint _vp_route_step_height_bottom = 0;
+SubSprite _vp_route_step_subsprite;
```

Comment:
These all seem like JGRPP specific changes - they're unused.

Reply:
It was inevitable I'd miss some of those. Will fix.